From d5e2187519830ce8f56705205d79840a5e32e3ad Mon Sep 17 00:00:00 2001
From: Derek Foreman <derekf@osg.samsung.com>
Date: Fri, 21 Nov 2014 12:31:04 -0600
Subject: [PATCH 75/96] rpi-renderer: don't pass a box to
 pixman_region32_contains_point()

We don't care which box contained the point, so don't pass one in.

Signed-off-by: Derek Foreman <derekf@osg.samsung.com>
Reviewed-by: Bryce Harrington <b.harrington@samsung.com>
---
 src/rpi-renderer.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/rpi-renderer.c b/src/rpi-renderer.c
index 2feab63..94d0424 100644
--- a/src/rpi-renderer.c
+++ b/src/rpi-renderer.c
@@ -337,8 +337,7 @@ apply_opaque_region(struct wl_shm_buffer *buffer,
 	for (y = 0; y < height; y++) {
 		for (x = 0; x < width; x++) {
 			int i = y * stride / 4 + x;
-			pixman_box32_t box;
-			if (pixman_region32_contains_point (opaque_region, x, y, &box)) {
+			if (pixman_region32_contains_point (opaque_region, x, y, NULL)) {
 				dst[i] = src[i] | 0xff000000;
 			} else {
 				dst[i] = src[i];
-- 
2.1.0

